pkgs/applications/networking/browsers/chromium/README.md

```diff
@@ -39,6 +39,16 @@ update `upstream-info.json`. After updates it is important to test at least
 `nixosTests.chromium` (or basic manual testing) and `google-chrome` (which
 reuses `upstream-info.json`).
 
+Note: Due to the script downloading many large tarballs it might be
+necessary to adjust the available tempfs size (it defaults to 10% of the
+systems memory)
+
+```nix
+services.logind.extraConfig = ''
+  RuntimeDirectorySize=4G
+'';
```

Comment:
This should use systemd.services

Also can't we just point this to /tmp instead of /run?

Reply:
> This should use systemd.services

Whats the difference?

> Also can't we just point this to /tmp instead of /run?

The tmpfs used is determined by `nix-prefetch-url`, at least from the manpage it doesn't look like it can be changed

Reply:
> Whats the difference?

I thought it was the same, but apparently it is not https://github.com/NixOS/nixpkgs/blob/nixos-unstable/nixos/modules/system/boot/systemd/logind.nix#L100